src/it/override-war-with-snapshot/invoker.properties

```diff
@@ -0,0 +1,12 @@
+# Pass the version of the hpi-plugin here and not in the pom, as pom filtering requires aggregator/child relationship
+
+# First just build the plugin that we depend on
+invoker.goals.1=-f war-with-plugin -ntp -Dhpi-plugin.version=${project.version} clean install -P quick-build -Drevision=1.0.0-SNAPSHOT
+# Then build another version and create a war containing the newer version
+invoker.goals.2=-f war-with-plugin -ntp -Dhpi-plugin.version=${project.version} clean install -P quick-build -P build-war -Drevision=2.0.0-SNAPSHOT
```

Comment:
slightly abnormal syntax of profile activation as invoker will split into multiple invocations on seeing a `,` so we use `-P` twice